install

```diff
@@ -645,10 +670,13 @@ else
 fi
 
 # Enable Apache modules
-if [ "$dist" != "centos" ]; then
+if [ "$dist" != "centos" ] && [ "$code" != "xenial" ]; then
     loudCmd "a2enmod rewrite php5"
+elif [ "$dist" = "ubuntu" ] && [ "$code" = "xenial" ]; then
+	loudCmd "a2enmod rewrite"
+	loudCmd "a2enmod php7.0"
 fi
-
+exit
```

Comment:
It looks like the code is bailing out on this line. I'm assuming this is some left over debug code.